scripts/User_Management/grade_9_mode.py

```diff
@@ -0,0 +1,41 @@
+from scripts._utils import ssh, pi, utils
+
+host_themes = "pi-themes"
+tvs = ["1", "2", "3", "4"]
+
+TV_OFF_CMD = "echo standby 0 | cec-client -s -d 1"
+TV_ON_CMD = "echo on 0 | cec-client -s -d 1"
+
+
+def engage():
+    ssh_connection = ssh.SSH(host_themes, pi.username, pi.password)
+    ssh_connection.send_cmd("amixer sset 'Headphone',0 0%")
+
+    for tv_number in tvs:
```

Comment:
don't duplicate, call the method from elsewhere.  If it isn't a method, make it one so code can be DRY

Reply:
fixed as per [6612eae](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/60/commits/6612eae166341b92fd6439632f661f323301261d)